ruddock/modules/hassle/helpers.py

```diff
@@ -119,7 +119,7 @@ def get_rooms_remaining():
   Gets the number of rooms remaining for each alley.
   Returns a dict mapping alley to number of remaining rooms.
   """
-  alley_counts = dict(zip(alleys, [0] * len(alleys)))
+  alley_counts = dict(list(zip(alleys, [0] * len(alleys))))
```

Comment:
Yes, these are equivalent. `2to3` will convert any suitable generator function call (e.g. `map`, `zip`, etc.) to a list by calling `list` around it, creating functionally equivalent but maybe slightly space-intensive functions. 

Ultimately, I will include this in the commit because it is better safe than sorry, and besides I don't think the amount of alleys will ever be so high as to encounter a space limit...